packages/webcert/src/components/certificateList/CertificateList.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import styled from 'styled-components'
+import { getActivePatient } from '../../store/patient/patientSelectors'
+import { getCertificateTypes, setUserPreference } from '../../store/user/userActions'
+import { getUserPreference, selectCertificateTypes } from '../../store/user/userSelectors'
+import CertificateListRow from './CertificateListRow'
+import fileIcon from '../../images/fileIcon.svg'
+import { useHistory } from 'react-router-dom'
+import { createNewCertificate, updateCreatedCertificateId } from '../../store/certificate/certificateActions'
+import { getCertificateId } from '../../store/certificate/certificateSelectors'
+
+const byFavorite = (a: any, b: any): number => {
```

Comment:
Any should be avoided

Reply:
Yes, I changed it according to VS Code suggestion.

Reply:
I can't get anything else than any to work. Maybe it's not a big deal here?

Reply:
For it to work you will have to create a type or interface out of the object you're mapping to on line 84, e.g.
```typescript
interface CertificateTypeViewModel {
  certificateName: string
  certificateInfo: string
  id: string
  issuerTypeId: string
  favorite: boolean
  createDisabled: boolean
}
```

Reply:
Ah, great. I fix.